controllers/entriesController.js

```diff
@@ -0,0 +1,44 @@
+const JournalEntry = require('../models/JournalEntry');
+
+const addEntry = (req, res) => {
+  const {
+    userId, title, entryBody, isPrivate,
+  } = req.body;
+
+  JournalEntry.addJournalEntry(title, entryBody, isPrivate)
+    .then(() => JournalEntry.getLastCreatedEntry(userId))
+    .then(() => res.status(201))
```

Comment:
`res.status()` sets the header but doesn't compete the response cycle. See [section in docs](https://expressjs.com/en/5x/api.html#res.status)